src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -938,6 +938,66 @@ public static partial class Kernel32
+        /// The module must have been loaded by the calling process.
+        /// </summary>
+        /// <param name="dwFlags">
+        /// This parameter can be zero or one or more of the following values.
+        /// If the module's reference count is incremented, the caller must use the <see cref="FreeLibrary"/> function to decrement the reference count when the module handle is no longer needed.</param>
+        /// <param name="lpModuleName">The name of the loaded module (either a .dll or .exe file), or an address in the module (if <paramref name="dwFlags"/> is <see cref="GetModuleHandleExFlags.GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS"/>).</param>
+        /// <param name="phModule">
+        /// A handle to the specified module. If the function fails, this parameter is NULL.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, returns true. If the function fails, the returns false.
+        /// To get extended error information, see <see cref="GetLastError"/>.
+        /// </returns>
+        /// <remarks>
+        /// The handle returned is not global or inheritable. It cannot be duplicated or used by another process.
+        /// This function does not retrieve handles for modules that were loaded using the <see cref="LoadLibraryExFlags.LOAD_LIBRARY_AS_DATAFILE"/> flag.
+        /// </remarks>
+        [DllImport(nameof(Kernel32), SetLastError = true, CharSet = CharSet.Auto)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool GetModuleHandleEx(GetModuleHandleExFlags dwFlags, string lpModuleName, out SafeLibraryHandle phModule);
+
+        /// <summary>
+        /// Retrieves the address of an exported function or variable from the specified dynamic-link library (DLL).
+        /// </summary>
+        /// <param name="hModule">A handle to the DLL module that contains the function or variable. The <see cref="LoadLibrary"/>, <see cref="LoadLibraryEx"/>, or <see cref="GetModuleHandle"/> function returns this handle.</param>
+        /// <param name="procName">The function or variable name, or the function's ordinal value. If this parameter is an ordinal value, it must be in the low-order word; the high-order word must be zero.</param>
+        /// <returns>
+        /// If the function succeeds, the return value is the address of the exported function or variable.
+        /// If the function fails, the return value is NULL.To get extended error information, call <see cref="GetLastError"/>.
+        /// </returns>
+        /// <remarks>This function does not retrieve handles for modules that were loaded using the <see cref="LoadLibraryExFlags.LOAD_LIBRARY_AS_DATAFILE"/> flag.</remarks>
+        [DllImport(nameof(Kernel32), SetLastError = true, CharSet = CharSet.Ansi, ExactSpelling =true)]
+        public static extern IntPtr GetProcAddress(SafeObjectHandle hModule, string procName);
```

Comment:
Ohh. One more thing.

I am thinking of changing `IntPtr GetProcAddress(SafeObjectHandle hModule, string procName)` to `IntPtr GetProcAddress(SafeLibraryHandle hModule, string procName)`.

When I switched to the nuget PInvoke functions I noticed that I do a LoadModule and the get this value to put into GetProcAddress. Also SafeObjectHandle is not compatible with SafeLibraryHandle.

OK with that guys?